.gitignore

```diff
@@ -1 +1,2 @@
 docs/*.dia.autosave
+stem.sublime-workspace
```

Comment:
No, there are actually two files. One is sublime-project and the other is sublime-workspace. The first is useful for checking in as it contains some project settings, whereas the other is better ignored as it contains trivialities such as which files are currently opened.